src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/filter_ratios.js

```diff
@@ -20,17 +20,23 @@
 const filter = (metric) => metric.type === 'filter_ratio';
 import { bucketTransform } from '../../helpers/bucket_transform';
 import { overwrite } from '../../helpers';
+import { esQuery } from '../../../../../../data/server';
 
-export function ratios(req, panel, series) {
+export function ratios(req, panel, series, esQueryConfig, indexPatternObject) {
   return (next) => (doc) => {
+    const filters = !panel.ignore_global_filter ? req.payload.filters : [];
```

Comment:
I'm not sure that we should pass` ignore_global_filter` or `payload.filters` into buildEsQuery method. For me looks like that for numerator/denominator we should pass empty array

@timroes could you please correct me if I'm wrong